drivers/vcluster/vcluster.go

```diff
@@ -76,10 +126,25 @@ func DeleteVCluster(vclusterName string) error {
 	return nil
 }
 
+// ExecuteVClusterCommand executes any generic vCluster command
+func ExecuteVClusterCommand(args ...string) (string, error) {
+	cmd := exec.Command("vcluster", args...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("failed executing vcluster command with args %v: %w, Output: %s", args, err, output)
+	}
+	return string(output), nil
+}
+
+// TerminateVCluster Terminates Vcluster and runs it in its own context
+func (v *VCluster) TerminateVCluster() error {
+	_, err := ExecuteVClusterCommand("delete", v.Name)
+	return err
+}
+
 // CreateVCluster This method creates a vcluster. This requires vcluster.yaml saved in a specific location.
 func CreateVCluster(vclusterName string, absPath string) error {
-	cmd := exec.Command("vcluster", "create", vclusterName, "-f", absPath, "--connect=false")
-	err := cmd.Run()
+	_, err := ExecuteVClusterCommand("create", vclusterName, "-f", absPath, "--connect=false")
```

Comment:
If we use vcluster on top of Managed kubernetes cluster like `EKS anywhere` then by default storage class would be not portworx storage class, during this time you need to change the default storageclass as mentioned here
`kubectl patch storageclass px-db -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'`
Otherwise we cannot create vCluster.
https://portworx.atlassian.net/wiki/spaces/TA/pages/2754543945/Aera+-+vCluster+Use+case#vCluster-Installation

This comment is not related to this changes, but we should have this fix in future changes.

Reply:
Hi Santosh, noted and thanks. I am aware of this and will be taking care of this in my next PR. Along with downloading vcluster binary during Torpedo installation.